examples/evaluation_and_profiling/swe_bench/tests/test_iterative_predictor.py

```
@@ -0,0 +1,700 @@
+        mock_checkout.assert_called_once()
+
+    @pytest.mark.asyncio
+    async def test_clone_cleans_existing_path(self, tmp_path):
+        """Test that clone removes existing directory before cloning."""
+        from nat_swe_bench.predictors.predict_iterative.tools.git_tool import clone_repository
+
+        target_path = tmp_path / "repo"
+        target_path.mkdir()
+        (target_path / "existing_file.txt").write_text("old content")
+
+        with patch('nat_swe_bench.predictors.predict_iterative.tools.git_tool.Repo') as MockRepo:
```

Ruff (0.14.14)

Local variable `MockRepo` is assigned to but never used

Remove assignment to unused variable `MockRepo`

(F841)